lib/deprecation_toolkit/rspec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "deprecation_toolkit/rspec_plugin"
+require "deprecation_toolkit/rspec_hook"
+
+RSpec.configure do |config|
+  config.after do |example|
+    DeprecationToolkit::RSpecHook.trigger_deprecation_toolkit_behavior(example)
```

Comment:
Why not call directly `DeprecationToolkit::TestTriggerer.trigger_deprecation_toolkit_behavior(example)`?